angular-app/src/app/models/NMNetwork.ts

```diff
@@ -0,0 +1,16 @@
+export interface NMNetwork {
+  token_address: string;
```

Comment:
It seems like everywhere else camelCase is used. Any reason for breaking this here?

Reply:
The interface looks like this because the data comes directly "as is" from the endpoint, see https://github.com/raiden-network/status-page/blob/master/mock/data/network-info.json

The data already existed in that format in various variables but its structure was not specified. 

I specified the structure to get more useful error messages during debugging if the data doesn't fit, ie this is part of giving useful warnings if endpoint data doesn't fit the expected format.

Reply:
camelCase is generally the convention in javascript; it appears to be so typescript too.